lib/_stream_writable.js

```diff
@@ -728,5 +746,12 @@ Object.defineProperty(Writable.prototype, 'writableFinished', {
 Writable.prototype.destroy = destroyImpl.destroy;
 Writable.prototype._undestroy = destroyImpl.undestroy;
 Writable.prototype._destroy = function(err, cb) {
+  const state = this._writableState;
+  for (let entry = state.bufferedRequest; entry; entry = entry.next) {
+    entry.callback(new ERR_STREAM_DESTROYED('write'));
+  }
+  state.bufferedRequest = null;
+  state.lastBufferedRequest = null;
+  state.bufferedRequestCount = 0;
```

Comment:
On a second thought, I think we should not change this method. Specifically, `_destroy`  is going to be overridden. We should add this somewhere else.

Reply:
Not sure where that would be though. `destroy()`? Which also can be override...

Reply:
Tried moving it to `destroy()`. Maybe that is preferable?